### PR TITLE
VC5Decompressor: support GoPro HERO10/HERO11 CFA pattern

### PR DESCRIPTION
**Pull request by kmilos**
Should be a more performant variant of https://github.com/darktable-org/rawspeed/pull/308

Didn't bother w/ further generalization because of the 1 vs 2 vs many rule ;) (GPR codec seems to use only these 2 patterns)

### CODE REVIEW

File: src/librawspeed/decompressors/VC5Decompressor.cpp [hunk 1]
```diff
@@ -826,28 +826,54 @@ void VC5Decompressor::combineFinalLowpassBands() const noexcept {
...
+        int rg = lowbands1(row, col) - mid;
+        int bg = lowbands2(row, col) - mid;
+        int gd = lowbands3(row, col) - mid;
+
+        int r = gs + 2 * rg;
+        int b = gs + 2 * bg;
+        int g1 = gs + gd;
+        int g2 = gs - gd;
+
+        out(2 * row + 0, 2 * col + 0) = static_cast<uint16_t>(mVC5LogTable[g1]);
```
Comment (LebedevRI) [inline src/librawspeed/decompressors/VC5Decompressor.cpp:872]: I *really* love that the green's have stable relative order.

**Review by LebedevRI — COMMENTED** [review]
Thanks, i hate it.

**Comment by LebedevRI** [comment]
BTW, someone still needs to contribute a raw sample from GoPo HERO11 to https://raw.pixls.us/.

**Comment by kmilos** [comment]
Yeah, admittedly not the most elegant...

**Pull request merged**